build/ci/postInstall.js

```diff
@@ -76,5 +77,23 @@ function createJupyterKernelWithoutSerialization() {
     console.log(colors.green(destPath + ' file generated (by Jupyter VSC)'));
 }
 
+/**
+ * In order to generate random bytes on Windows without taking a dependency on native node modules
+ * which we then need to build xplat and bundle with the extension, download a prebuilt executable
+ * which directly consumes BCryptGenRandom in bcrypt.dll and outputs random bytes as hex. This
+ * executable is required for trusted notebooks key generation and is included with the built extension.
+ */
+function downloadBCryptGenRandomExecutable() {
+    console.log('Downloading BCryptGenRandom.exe...');
+    const executableName = 'BCryptGenRandom.exe';
+    const uri = `https://pvsc.blob.core.windows.net/extension-builds-juypter/${executableName}`;
+    const dest = path.resolve(path.dirname(__dirname), '..', 'src', 'BCryptGenRandom');
+    fs.ensureDirSync(dest);
+    const destination = path.join(dest, executableName);
+    cp.execSync(`curl --output ${destination} ${uri}`);
```

Comment:
Not everyone has `curl`, why not just use node API to download a file or use an npm package like `download` already used today.